onnxruntime/core/graph/model.cc

```diff
@@ -68,17 +68,10 @@ Model::Model(const std::string& graph_name,
     opset_id_proto->set_version(domain.second);
   }
 
-  std::unordered_map<std::string, const ONNX_NAMESPACE::FunctionProto*> model_functions_map;
-  for (auto& func : model_functions) {
-    auto func_ptr = model_proto_.add_functions();
-    func_ptr->CopyFrom(func);
-    model_functions_map[func_ptr->name()] = func_ptr;
-  }
-
   // need to call private ctor so can't use make_shared
   GSL_SUPPRESS(r .11)
   graph_.reset(new Graph(*this, model_proto_.mutable_graph(), *p_domain_to_version, IrVersion(), schema_registry,
-                         logger, model_functions_map));
+                         logger, {}));
```

Comment:
model_functions_map is also used in graph.cc during resolve. It needs to be removed from there too. See :
https://github.com/microsoft/onnxruntime/blob/master/onnxruntime/core/graph/graph.cc#L2003